### PR TITLE
Fixes #272

### LINKED ISSUE #272: left sidebar: does not support long section names
If section name does not fit into one line it hides another section that follows it.

### PR DESCRIPTION
**Pull request by ProjGoodF**
Fixes #272. The best option is to use jquery.slideToggle, since this function can calculate the height of the element in the case of height: auto

### CODE REVIEW

File: src/Documentation/SidebarMenu/SidebarMenu.js [hunk 1]
```diff
@@ -9,6 +10,24 @@ import styled from 'styled-components'
 import { media, OnlyDesktop } from '../../styles'
 
 export default class SidebarMenu extends React.Component {
+    constructor(props) {
+        super(props);
+        this.collapse = this.collapse.bind(this);
+    }
+    collapse(){
+        setTimeout(function () {
+            $('[data-open=true]').slideDown();
```
Comment (shcheklein) [inline src/Documentation/SidebarMenu/SidebarMenu.js]: please, let's use 2 spaces as in other code

Comment (ProjGoodF) [reply src/Documentation/SidebarMenu/SidebarMenu.js]: Ok, there were 4 spaces in the source code

Comment (shcheklein) [reply src/Documentation/SidebarMenu/SidebarMenu.js]: Yep. Looks there was a mix. Let's change it please, so that we have it consistent. Thanks!

**Comment by shcheklein** [comment]
I'm getting a warning in the logs:

```Warning: Each child in an array or iterator should have a unique "key" prop.

Check the top-level render call using <styled.div>. See https://fb.me/react-warning-keys for more information.
    in React.Fragment
    in SidebarMenu
    in div
    in styled.div
    in div
    in styled.div
    in div
    in styled.div
    in div
    in styled.div
    in Layout
    in withRouter(Layout)
    in Page
    in Documentation
    in StyleSheetManager
    in Unknown
    in Container
    in App
    in Capture```

**Pull request merged**